src/pages/App.tsx

```diff
@@ -65,29 +65,18 @@ initializeAnalytics(ANALYTICS_DUMMY_KEY, OriginApplication.INTERFACE, {
   isProductionEnv: isProductionEnv(),
 })
 
-const AppWrapper = styled.div`
```

Comment:
We don't need to set a `height: 100%` on a top-level container at all. This was also causing a bottom overflow caused by a `position: sticky` of an inner `MobileBottomBar` component + `BodyWrapper` having a `height: 100%` either.